### PR TITLE
Release 2.2.1

### PR DESCRIPTION
**Pull request by HelenCampbell**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+##Supported Release 2.2.1
+###Summary
+
+Small release for support of newer Puppet versions.
```
Comment (daenney) [inline CHANGELOG.md]: This is by no means an adequate change log. We should actually state what's changed/fixed. The generic "bug fixes and improvement" changelogs are incredibly infuriating for anyone that actually needs to figure out what changes made it in when.

**Comment by HelenCampbell** [comment]
Hey @daenney , There is nothing else changing with this release, it is just a version bump for pe. Also we collectively decided to take out the date as it didn't always match what was in git or the forge.

**Comment by daenney** [comment]
So, we're just releasing a new version of the module with no changes at all? If there are changes, then what are they? And if it's specifically to support something in PE, we should mention that.

**Comment by bmjen** [comment]
@daenney as the changelog states, this is a Y release to bump the Puppet Enterprise upper bound in the metadata.json to support the upcoming 2015.3.0. Due to the new versioning scheme, we need to figure out a better way to handle upcoming versions of PE, until then.. we are merely releasing the previous version + metadata bump.

**Comment by daenney** [comment]
@bmjen It doesn't state that. It literally says:

> Small release for support of newer Puppet versions.

**Comment by tphoney** [comment]
@daenney, this was the format for the last release of PE, does the following sound better. bare in mind we now need to update 20+ modules again

##Supported Release 2.2.1
###Summary

Small release for support of newer PE versions. This increments the version of PE in the metadata.json file

**Comment by daenney** [comment]
:+1:

**Comment by tphoney** [comment]
np, we will do this now :)

**Comment by bmjen** [comment]
Thanks @daenney!

**Comment by tphoney** [comment]
@daenney  All the affected modules now have this text.

**Comment by daenney** [comment]
Woo! :100:

**Pull request merged**